chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/limits/JLimit.java

```diff
@@ -0,0 +1,150 @@
+package com.griddynamics.jagger.user.test.configurations.limits;
```

Comment:
This class looks wrong
1. metricName is mandatory. It may be custom or one from JFG-981 
2. We have 2 different types of limits - relative to static value and baseline. I think we need to classes for this limits like for JLoadProfile or JTerminationCriteria. 
- one of them will not have refval
- for another refval will be mandatory
3. Thresholds are mandatory, but this version of builder allows create limits without thresholds
`public static Builder builder()`